src/test_c2s_srv.c

```diff
@@ -354,15 +369,19 @@ int test_c2s(int ctlsockfd, tcp_stat_agent* agent, TestOptions* testOptions,
     sel_tv.tv_sec = 11;  // time out after 11 seconds
     sel_tv.tv_usec = 0;
     FD_ZERO(&rfd);
-    FD_SET(recvsfd, &rfd);
+    FD_SET(c2s_conn.socket, &rfd);
     for (;;) {
-      msgretvalue = select(recvsfd + 1, &rfd, NULL, NULL, &sel_tv);
+      msgretvalue = select(c2s_conn.socket + 1, &rfd, NULL, NULL, &sel_tv);
       if ((msgretvalue == -1) && (errno == EINTR)) {
         // socket interrupted. Continue waiting for activity on socket
         continue;
       }
       if (msgretvalue > 0) {  // read from socket
-        tmpbytecount = read(recvsfd, buff, sizeof(buff));
+        if (c2s_conn.ssl == NULL) {
```

Comment:
It seems like in the other CLs the check was whether `ssl != NULL`. Easier to read if we stay consistent.